blogposts/climate-transparency-report-2023.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Climate Transparency Report 2023 🌳'
+date: '2024-06-26T00:00:00'
+tags: ['Transparency', 'Carbon', 'Climate']
+author: 'Keno Dreßel'
+previewText: 'Our annual report showing our footprint aggregated in categories.'
+previewImage: '/blogposts/preview/climate-transparency-report-2023.png'
+---
+
+<p align="center">
+<img src="/blogposts/preview/climate-transparency-report-2023.png" alt="green circle with green foot inside, CO<sub>2</sub> is written on top and two arrows on the side point downwards" width="300"/>
+<a href="https://www.flaticon.com/free-icons/carbon" title="carbon icons">Carbon icons created by Flat Icons - Flaticon</a>
+</p>
+
+## <span style={{ color: '#10AABF' }}>Our Motivation</span>
+
+At Unit 214 GmbH, we prioritize environmental sustainability, as evident in our efforts to minimize carbon emissions.
+Using euros and CO<sub>2</sub>e kg conversion metrics, this report outlines our CO<sub>2</sub>e emissions for 2023, totaling 17.310 kg, and underscores our commitment to reducing our carbon footprint.
+In this report, we focus on the major emission items and acknowledge additional categories contributing approximately 3.484 kg CO<sub>2</sub>e.
+The main categories are:
```

Comment:
I know I wrote the report but shoud we already mention 2022 here?

Reply:
> I know I wrote the report but shoud we already mention 2022 here?

@kenodressel , I am not able to follow. Could you please help me understand what you mean?